src/ECharts/Types.purs

```diff
@@ -178,11 +174,11 @@ newtype Item = Item Foreign
 newtype Coord = Coord Foreign
 
 coord ∷ Number → Number → Coord
-coord x y = Coord $ toForeign [ x, y ]
+coord x y = Coord $ unsafeToForeign [ x, y ]
 
 type LegendEventR =
   { name ∷ String
-  , selected ∷ SM.StrMap Boolean
+  , selected ∷ Set Boolean
```

Comment:
Was this changed occasionally? `Set Boolean` definitely has different semantics :smile: